docs/misconfiguration/policy/custom.md

```diff
@@ -0,0 +1,180 @@
+# Custom Policies
+
+## Overview
+You can write custom policies in [Rego](https://www.openpolicyagent.org/docs/latest/policy-language/).
+Once you fisnish writing custom policies, you can pass the directory where those policies are stored with `--policy` option.
+
+``` bash
+trivy conf --policy /path/to/custom_policies --namespaces user /path/to/config_dir
+```
+
+As for `--namespaces` option, the detail is described as below.
+
+### General file formats
+If a file name matches the following file patterns, Trivy will parse the file and pass it as input to your Rego policy.
+
+| File format    | File pattern                                     |
+| -------------- | ------------------------------------------------ |
+| JSON           | `*.json`                                         |
+| YAML           | `*.yaml`                                         |
+| TOML           | `*.toml`                                         |
+| HCL            | `*.hcl`, `*.hcl1`, and `*.hcl2`                  |
+| Dockerfile     | `Dockerfile`, `Dockerfile.*`, and `*.Dockerfile` |
+
+### IaC file formats
+In the above general file formats, Trivy automatically identifies the following types of configuration files:
```

Comment:
maybe just to distinguish this list from the previous list, call the previous one "file formats" and this one "configuration languages"

Reply:
done https://github.com/aquasecurity/trivy/pull/1101/commits/f3ec1597ad1a9f26c0d7ab7f3f80b27bdd5de2f2